api/src/public/login/style.css

```diff
@@ -7,7 +7,7 @@ body {
   margin: 0;
 }
 body, input {
-  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-family: Noto, sans-serif;
```

Comment:
This won't work if the device doesn't already have Noto installed. For everyone else you'll need to link to the Noto source, eg: https://github.com/medic/cht-core/blob/master/webapp/src/css/font.less#L3-L7

Reply:
ok @garethbowen, I linked it first, then I tried to remove it and it worked too, so I thought it not to be needed, now I get it, thank you.